src/scripts/index.js

```diff
@@ -27,146 +24,170 @@ class Calculator {
   createEventListener() {
     let target = event.target;
 
-    if (target.classList[0] === 'number' && document.getElementById(`output-screen_${this.calculatorId}`).textContent.length < 12) {
+    if (target.classList[0] === 'number' && this.oneNumber.length < 12) {
         this.pressNumberAction(target, this.calculatorId);
```

Comment:
Can we use contains() method of classList?

Reply:
You are right, using of this methos is more logical. I will use it.